packages/manager/modules/manager-pci-common/src/components/flavor-selector/FlavorTile.component.tsx

```diff
@@ -195,7 +195,7 @@ export function FlavorTile({
           )}
         </div>
         <hr className={separatorClass} />
-        {flavorPrice.monthly && (
+        {flavorPrice.monthly > 0 && (
```

Comment:
Doing this means that you never want to display prices with 0 value ? (e.g. 0€) Is it the purpose?